content/articles/Website-vs-Web-app/index.md

```diff
@@ -0,0 +1,106 @@
+5. `Digitalized`- You want to ensure that your website uses the latest technological trends. You do not want to construct a website not supporting the needs of today.
+6. `Relevant Content` - The content in the website should be relevant to the users. For example, you cannot post cars on a job board website. When your website is a work board it should be provided with information and tips on how to conduct an interview for instance.
+
+#### Importance of a website
+
+1. ``Updates to customers``- As your website is available 24/7, updates and notices can be easily published for your customers. This is a way to keep them up-to-date on all you do. When something matters particularly to you, you are more likely to upsell it.
+
+2. `Professionalism` - Studies show that most people opt to deal with businesses with websites compared to those that lack one. 
+
+3. ``Creates a large market pool``- Your business accessibility is exponentially increasing through a website, as it is accessible from anywhere in the world. Websites, therefore, help companies to break geographical obstacles and increase sales significantly.
+4. ``Growth Opportunity``- A website may serve as a portfolio of a company or resume. This can prove to be beneficial for companies as the site can act as a reference for potential investors who know what you have achieved already and your plans for the future as a company.
+
+### Web application
+A web application is a program that can be accessed via a browser and is linked to a database. A web application contains both functional and interactive elements.
+Unlike websites, web apps are more complex to create (because they are very customized) and require an experienced team.
+Below are examples of web application:
+- [Twitter](https://twitter.com/?lang=en)
+- [Gmail](https://mail.google.com/)
+- [Wikipedia](https://www.wikipedia.org/)
+
+#### Characteristics of A web Application
+1. `Accessibility`: A good web app should available and usable by as many people as possible.
+2. `Storage system` -A web app should have a good and reliable database for storing the developers' and end-users information.
+3. ``Easy to update``- Business keeps on expanding as time passes by. Therefore a web app should be simple to update.
+4. `Cross-platform`- A web app should be able to run on most if not all operating systems using different browsers.
+
+#### Importance of a Web Application
+
+1. `Security`: Web applications ensure sites and programs are secure since they are designed with more privacy and a high level of protection.
+2. `Easy to use`: End users can access them from any device at any time. 
```

Comment:
This is not an importance. I think it looks more like an advantage. How about you talk about it brings ease to most daily activities such as communication, internet banking, etc.